core/object.go

```diff
@@ -522,12 +523,22 @@ func (mw *Middleware) ObjectListSetObjectType(cctx context.Context, req *pb.RpcO
 	}
 
 	if err := mw.doBlockService(func(bs *block.Service) (err error) {
+		var (
+			mErr       multierror.Error
+			anySucceed bool
+		)
 		for _, objID := range req.ObjectIds {
 			if err = bs.SetObjectTypes(ctx, objID, []string{req.ObjectTypeId}); err != nil {
-				return err
+				log.Errorf("failed to set object type to object '%s': %v", objID, err)
```

Comment:
move objID to separate log field `objectID`

Reply:
Thank you! Forgot about our log fields )